shared/general/container-ssh.md

```diff
@@ -1,46 +1,83 @@
-To help you debug and develop your applications, we've provided a browser based terminal and a command line tool for easy SSH access to your devices. With these tools, you have console access to any of your running containers, as well as to the host OS, letting you test out small snippets of code and check system logs on your device.
+To help you debug and develop your applications, we've provided a browser-based terminal and a command line tool for easy SSH access to your devices. With these tools, you have console access to any of your running containers, as well as to the host OS, letting you test out small snippets of code and check system logs on your device. You can also access your device via a standalone SSH client.
 
 __Note:__ Host OS SSH access is available for devices running {{ $names.os.lower }} version 2.7.5 and above.
 
 ## Using the dashboard web terminal
 
 To use this feature, navigate to your application and select the device you want to access. You will see a *Terminal* window below the *Logs* window:
 
-<img src="/img/common/device/terminal.png" width="60%">
-
-In order to start a terminal session for a service, you first need to ensure that your device is online and that the service container is running. If the container code crashes or ends quickly, it is not possible to attach a console to it. Check out our [Dockerfile][dockerfile-init] and [Docker Compose][compose-init] guides for more information on keeping your containers running.
```

Comment:
Right, this was an over-aggressive purge of outdated init links! Reworded this and avoided another note :) I actually had this issue testing this as many of our hello world projects simply print "hello world" and exit rather than looping/sleeping.